minio.json

```
@@ -0,0 +1,54 @@
+{
+    "MinIO": {
+        "description": "MinIO is an S3-compatible object storage server.  Requires Rockstor 3.9.2-39 or later.<p>Based on the official MinIO docker image: <a href='https://hub.docker.com/r/minio/minio' target='_blank'>https://hub.docker.com/r/minio/minio</a>.",
```

Detail architecture supported:
```suggestion
        "description": "MinIO is an S3-compatible object storage server. This rock-on requires Rockstor 3.9.2-39 or later and is compatible with x86_64/amd64 machines only.<p>Based on the official MinIO docker image: <a href='https://hub.docker.com/r/minio/minio' target='_blank'>https://hub.docker.com/r/minio/minio</a>.",
```